contracts/ragnarok/ManagedPortfolio.sol

```diff
@@ -22,17 +25,23 @@ contract ManagedPortfolio is IERC721Receiver, ERC20, Ownable {
     IERC20WithDecimals public underlyingToken;
     BulletLoans public bulletLoans;
     uint256 public endDate;
+    BIP public managerFee;
+    address public manager;
 
     event BulletLoanCreated(uint256 id);
 
     constructor(
         IERC20WithDecimals _underlyingToken,
         BulletLoans _bulletLoans,
-        uint256 _duration
+        uint256 _duration,
+        BIP _managerFee,
+        address _manager
```

Comment:
I feel like we assumed that `manager` is just the `msg.sender` (the wallet that creates the portfolio). But perhaps it's better to make it explicit.